microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color.md

```diff
@@ -60,10 +60,219 @@ Each icon in the `icons` array should come with at least the `src` and `sizes` p
 
 Learn more about [the icons member](https://developer.mozilla.org/docs/Web/Manifest/icons).
 
-<!-- TODO: add information about which sizes Windows require at a minimum, and which sizes are used where -->
+## Icon Image Sizes
 
+Your PWA can be enhanced on Windows by supplying the images with specific dimensions in your web app manifest. We recommend choosing one of the options below:
+
+* Level 1: (Basic image support: 512x512) This is the base image from which to generate missing images
+* Level 2: (Titles) At this level, your web app manifest contains tile images for the default (1x) display scale. Each of the images must be in PNG format and purpose any.Here is a list of recommended images and sizes.
+    * 44x44 - app icon
+    * 71x71 - small tile
+    * 150x150 - medium tile
+    * 350x150 - wide tile
+    * 310x310 - large tile
+    * 50x50 - store logo
+    * 620x300 - splash screen
+* Level 3: (Tiles with display scales) At this level, your web app manifest icons should include images with these dimensions:
+    * 44x44 - app icon
+    * 55x55 - app icon 1.25x display scale
+    * 66x66 - app icon 1.5x display scale
+    * 88x88 - app icon 2x display scale
+    * 176x176 - app icon 4x display scale
+    * 71x71 - small tile
+    * 89x89 - small tile 1.25x display scale
+    * 107x107 - small tile 1.5x display scale
+    * 142x142 - small tile 2x display scale
+    * 284x284 - small tile 4x display scale
+    * 150x150 - medium tile
+    * 188x188 - medium tile 1.25x display scale
+    * 225x225 - medium tile 1.5x display scale
+    * 300x300 - medium tile 2x display scale
+    * 600x600 - medium tile 4x display scale
+    * 350x150 - wide tile
+    * 388x188 - wide tile 1.25x display scale
+    * 465x225 - wide tile 1.5x display scale
+    * 620x300 - wide tile 2x display scale
+    * 1240x600 - wide tile 4x display scale
+    * 310x310 - large tile
+    * 388x388 - large tile 1.25x display scale
+    * 465x465 - large tile 1.5x display scale
+    * 620x620 - large tile 2x display scale
+    * 1240x1240 - large tile 4x display scale
+    * 50x50 - store tile
+    * 63x63 - store tile 1.25x display scale
+    * 75x75 - store tile 1.5x display scale
+    * 100x100 - store tile 2x display scale
+    * 200x200 - store tile 4x display scale
+    * 620x300 - splash screen
+    * 775x375 - splash screen 1.25x display scale  
+    * 930x450 - splash screen 1.5x display scale
+    * 1240x600 - splash screen 2x display scale
+    * 2480x1200 - splash screen 4x display scale
+
+For level 3, your web app manifest should contain tile images for all Windows display scale sizes.
+
+Display scales are user-configurable in Windows:
+![Display Sample](../media/windows-image-display-scales.png) 
```

Comment:
And then you can remove this part since it's now above. Also, I don't think this image is helping much here. Just saying that the scale is configurable is enough, this article doesn't need to explain where to change it.

Reply:
removed Display scales are user-configurable in Windows:
![Display Sample](../media/windows-image-display-scales.png)